Formula/tmux.rb

```diff
@@ -40,6 +39,11 @@ class Tmux < Formula
   depends_on "pkg-config" => :build
   depends_on "libevent"
 
+  resource "completion" do
+    url "https://raw.githubusercontent.com/przepompownia/tmux-bash-completion/v0.0.1/completions/tmux"
```

Comment:
It is officially recommended by tmux: https://github.com/tmux/tmux/blob/master/README